docs/varnish/vcl/varnish5.vcl

```diff
@@ -127,32 +127,33 @@ sub ez_purge {
     // Retrieve purge token, needs to be here due to restart, match for PURGE method done within
     call ez_invalidate_token;
```

Comment:
Not a regression from this PR, however the code in this subroutine will need to be adapted to also take into account `req.method == "PURGEKEYS"`.

Ref: `if (req.restarts == 0 && req.method == "PURGE" && req.http.x-invalidate-token) {`

Reply:
Proposal for fix on that in e377e19